lib/galaxy/webapps/galaxy/buildapp.py

```diff
@@ -285,14 +285,14 @@ def postfork_setup():
 def populate_api_routes(webapp, app):
     webapp.add_api_controllers("galaxy.webapps.galaxy.api", app)
 
-    webapp.mapper.resource(
-        "user",
-        "users",
```

Comment:
So this means we're mapping both `user` and `users`, so
```
curl -v -X 'DELETE' 'http://127.0.0.1:8080/api/groups/any-group-id/user/any-user-id' -H 'accept: application/json'
```
and 
```
curl -v -X 'DELETE' 'http://127.0.0.1:8080/api/groups/any-group-id/users/any-user-id' -H 'accept: application/json'
```
used to work.

Reply:
Got it. Thanks!!